src/components/Offers.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Col, Row } from 'reactstrap';
+import OfferDetails from './OfferDetails';
+
+function Offers({ offers }) {
+  return (
+    <Row>
+      {offers && offers.map((offer) => (
+        <Col sm={12} md={6} lg={4} key={offer._id}>
+          <OfferDetails offer={offer} />
+        </Col>
+      ))}
+    </Row>
+  );
+}
+
+Offers.propTypes = {};
```

Comment:
You are missing the PropTypes validation here

Reply:
Fixed